merlin/schema/schema.py

```diff
@@ -297,13 +297,17 @@ def value_count(self) -> Optional[Domain]:
         value_count = self.properties.get("value_count")
         return Domain(**value_count) if value_count else None
 
-    def __merge__(self, other):
+    def __or__(self, other):
```

Comment:
Is this roughly equivalent in purpose to the `__add__` on the Schema level? if so, maybe would be clearer to align both the Schema and ColumnSchema with a similar API for combining?

Reply:
>  addition doesn't imply that either side gets priority in the same way that or/union do.

The implementation of Schema addition calls this ColumnSchema merge method, which suggests that the current sche,a addition does depend on the order. Making it a union operator would be more consistent if that's what it's intended to do.

Reply:
Agreed